src/Core/Core.csproj

```diff
@@ -24,8 +24,8 @@
     <PackageReference Include="AWSSDK.SimpleEmail" Version="3.7.0.150" />
     <PackageReference Include="AWSSDK.SQS" Version="3.7.2.47" />
     <PackageReference Include="Azure.Extensions.AspNetCore.DataProtection.Blobs" Version="1.2.1" />
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.11.0" />
-    <PackageReference Include="Azure.Storage.Queues" Version="12.9.0" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.14.1" />
+    <PackageReference Include="Azure.Storage.Queues" Version="12.12.0" />
```

Comment:
There should be `package.lock.json` updates to go along with these changes. To generate them you can either

1. Revert these changes and update the package by running `nuget add package Azure.Storage.Blobs` (and again for Queues)
2. Delete the `obj` directory found at `src/Core/obj` and running `dotnet restore`

Reply:
@MGibson1, Oh, now I see. Will do this today. Thanks for the review!